metadata_backend/helpers/schemas/BP.schema.xsd

```diff
@@ -3,9 +3,11 @@
     <xs:import schemaLocation="BP.annotation.xsd"/>
     <xs:import schemaLocation="BP.dataset.xsd"/>
     <xs:import schemaLocation="BP.image.xsd"/>
+    <xs:import schemaLocation="BP.observation.xsd"/>
     <xs:import schemaLocation="BP.sample.xsd"/>
+    <xs:import schemaLocation="BP.staining.xsd"/>
```

Comment:
I don't see either of the added imports in the repo. Is it supposed to be like this?

Reply:
Will be added in #644 and #645